lib/_tls_common.js

```diff
@@ -153,6 +153,15 @@ exports.createSecureContext = function createSecureContext(options) {
     }
   }
 
+  const sigalgs = options.sigalgs;
+  if (sigalgs) {
```

Comment:
This allows `0`, `''`, and `false` to be a sigalg, I think only `null` and `undefined` should be allowed to mean "no sigalgs".

Reply:
Sorry, don't quite get it. `0` `''` and `false` would be ignored, leaving us with the default signature algorithm list. We could of course let it pass through, but `0` and `false` would throw just on the next line, and `''` I believe would not be accepted by OpenSSL.